src/NetscriptFunctions.ts

```diff
@@ -1072,13 +1073,32 @@ const base: InternalAPI<NS> = {
       levelCost: Player.mults.hacknet_node_level_cost,
     };
   },
-  getBitNodeMultipliers: (ctx: NetscriptContext) => (): IBNMults => {
-    if (Player.sourceFileLvl(5) <= 0 && Player.bitNodeN !== 5) {
-      throw helpers.makeRuntimeErrorMsg(ctx, "Requires Source-File 5 to run.");
-    }
-    const copy = Object.assign({}, BitNodeMultipliers);
-    return copy;
-  },
+  getBitNodeMultipliers:
+    (ctx: NetscriptContext) =>
+    (_bitNode: unknown = Player.bitNodeN, _lvl: unknown = 1): IBNMults => {
```

Comment:
This default _lvl = 1 will result in an issue if the player is in a later iteration of BN12 and tries running getBitNodeMultipliers() with no arguments. Player would expect their current multipliers, they will instead only see the multipliers for 12.1

Reply:
resolved on lines 1097-1100